src/components/landing/subjects/subject-landing-featured.tsx

```diff
@@ -93,8 +93,6 @@ export function SubjectLandingFeatured({
 
   function renderTypeIcon(type: string) {
     const icon = getIconByTypename(type)
-    return (
-      <FontAwesomeIcon icon={icon} title={getTranslatedType(strings, type)} />
-    )
+    return <FaIcon icon={icon} />
```

Comment:
removed title

Reply:
I have removed the title here because the type string is visible directly next to it and a title would just be a duplicate

Reply:
Check. Thanks for the explanation.